sdk/identity/azcertificatemanager/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/template/identity/azcertificatemanager
```

Comment:
Path needs to match directory structure, so sdk/identity/azcertificatemanager.

Comment applies to all paths in this document.